curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/american-british-translator.md

```diff
@@ -11,11 +11,21 @@ Build a full stack JavaScript app that is functionally similar to this: <a href=
 Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.
 
 Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-american-british-english-translator">this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-american-british-english-translator/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+
+<h3>Instructions</h3>
+
+
```

Comment:
```suggestion
```

I feel like we don't need this

Reply:
This is part of the Project Markdown Boilerplate, as near as I can tell.  I didn't put it there, it was already there.  I'm disinclined to remove it unless that's the standard.

Reply:
I just realized it was a duplicated header.  I think it was a "cut and paste error".